sources/packages/web/src/components/common/Reports.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <full-page-container>
```

Comment:
`<full-page-container>` hardly will be added at the component level. Are we sure about having it shared at this level?

Reply:
@dheepak-aot mentioned it as well when we were debugging the code but I missed to change it. 😅 Will move it outside.

Reply:
Not a blocker, for this component and this feature it may not be a problem but even the `body-header-container` should be avoid sometimes.

Reply:
Removed it.